co2calculator/distances.py

```diff
@@ -52,6 +57,17 @@ class DistanceRequest(BaseModel):
     destination: Union[StructuredLocation, TrainStation, Airport]
 
 
+class Coordinate(BaseModel):
+    lat: confloat(ge=-90, le=90)
+    long: confloat(ge=-180, le=180)
+    lat_rad: confloat(ge=-np.pi / 2, le=np.pi / 2) = None
+    long_rad: confloat(ge=-np.pi, le=np.pi) = None
+
+    def deg2rad(self):
+        self.lat_rad = np.deg2rad(self.lat)
+        self.long_rad = np.deg2rad(self.long)
+
+
 # Module's exceptions
 class InvalidSpatialInput(Exception):
```

Comment:
Are there more special exceptions? Then we could put them in a separate module called `exceptions.py`